### DISCUSSION TITLE
timestamp for YouTube

### DISCUSSION THREAD

**Discussion by ghost** [community/community#106693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

is it possible to add timestamp as an option for YouTube so we copy video at certain time when we copied them if we wish

thanks

**Comment by rafidRD** [comment]
Hey if you mark mine I will mark yours in return... deal?

**Comment by rafidRD** [comment]
**try automation with selenium**

**Comment by gil-son** [comment]
1. Copy the data of the video 
2. Convert the data to timestamp into your application
